### DISCUSSION TITLE
微信小程序模拟失败

### DISCUSSION THREAD

**Discussion by Albert-ll** [Mikotwa/FuckLocation#29] [Q&A] [answered]
微信发送位置和共享位置都可以模拟，小程序定位不行。发现修改软件设置中的ECN/NCI和Tac，可以模拟小程序。但不知道怎么填想要的位置参数

**Comment by Mikotwa** [comment, marked as answer]
> 发现修改软件设置中的 ECN/NCI 和 Tac，可以模拟小程序。但不知道怎么填想要的位置参数

请阅读本软件的 README.md。我们不解答任何有关如何填写位置参数的问题
